src/inputs/input/input.jsx

```diff
@@ -12,10 +12,31 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import Icon from '../../dataDisplay/icon';
+import {
+    DOT_KEY_CODE,
+    MINUS_KEY_CODE,
+} from '../../global/constants';
 
 const propTypes = {
+    /**
+     * Indicates whether the value of the control can be automatically completed by the browser.
+     */
     autoComplete: PropTypes.oneOf(['off', 'on']),
+    /**
+     * Indicates field can have decimal value for input type number
+     */
+    allowDecimals: PropTypes.bool,
+    /**
+     * Gives Input immediate focus.
+     */
     autoFocus: PropTypes.bool,
+    /**
+     * Indicates field can have decimal value for input type number
```

Comment:
Oops!  Copy-and-past error.
Should say:
```
Indicates whether or not the field allows negative numbers (or just positive numbers) for <Input> of type "number".
```

Reply:
Please check this commit https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/8429b8ffd987325a6ae504db3aa2341b60de60ef